core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -472,7 +473,15 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
 
       for {
         _ <- Future.traverse(policies) { policy =>
-          samDAO.addUserToPolicy(SamResourceTypeNames.billingProject, projectName.value, policy, projectAccessUpdate.email, userInfo)
+          samDAO.addUserToPolicy(SamResourceTypeNames.billingProject, projectName.value, policy, projectAccessUpdate.email, userInfo).recoverWith {
+            case regrets: Throwable =>
+              if (policy == SamBillingProjectPolicyNames.canComputeUser) {
+                logger.info(s"error adding user to canComputeUser policy for $projectName because it is a v2 project. regrets: ${regrets.getMessage}")
```

Comment:
I'm a bit worried this could fail for other reasons but will say it failed due to being a v2 billing project. Could we check that the status in the error is 404 (or whatever it returns) to make it more precise?
(Or at the very least change the error message to say that the error was "... *probably* because ...")

Reply:
```suggestion
              if (policy == SamBillingProjectPolicyNames.canComputeUser) {
                logger.info(s"error adding user to canComputeUser policy for $projectName because it is a v2 billing project. regrets: ${regrets.getMessage}")
```

also suggest the same change below.